website/fiddle/public/wc/luigiExampleWC.js

```diff
@@ -0,0 +1,265 @@
+                            </div>
+                            <div class="fddocs-container" style="float: left; padding-left:10px;padding-top: 8px;">
+                                <button class="fd-button fd-button--positive" id="bOpenDialog" style="width:150px;">Open Dialog</button>
+                            </div>
+                          </div>
+                      </div>
+                      <div style="width:100%;float: left;">
+                          <div style="padding-top:5px;">
+                            <div style="float: left;padding-top: 10px;">
+                              <fundamental-input id="splitUrl" defaultValue="/home/wc1" 
+                              label="Insert url to be displayed" width="250px" labelWidth="160px"/>
+                            </div>
+                            <div class="fddocs-container" style="float: left; padding-left:10px;padding-top: 8px;">
+                                <button class="fd-button fd-button--positive" id="bSplitDialog" style="width:150px;">Open Split</button>
+                            </div>
+                          </div>
+                      </div>
+                      <div style="width:100%;float: left;">
+                          <div style="padding-top:5px;">
+                            <div style="float: left;padding-top: 10px;">
+                              <fundamental-input id="drawerUrl" defaultValue="/home/tree" 
+                              label="Insert url to be displayed" width="250px" labelWidth="160px"/>
+                            </div>
+                            <div class="fddocs-container" style="float: left; padding-left:10px;padding-top: 8px;">
+                                <button class="fd-button fd-button--positive" id="bDrawer" style="width:150px;">Open Drawer</button>
+                            </div>
+                          </div>
+                      </div>
+                      
+                      <div style="width:100%;float: left;">
+                          <div style="padding-top:5px;">
+                            <div style="float: left;padding-top: 10px;">
+                              <fundamental-input id="navigationUrl" defaultValue="/home/empty" 
+                              label="Insert url to go" width="250px" labelWidth="160px"/>
+                            </div>
+                            <div class="fddocs-container" style="float: left; padding-left:10px;padding-top: 8px;">
+                                <button class="fd-button fd-button--positive" id="bNavigation" style="width:150px;">Navigate</button>
+                            </div>
+                          </div>
+                      </div>
+                      
+                </div>
+              </div>
+            </div>
+            <br />
+        </div>
+        <footer>
+          
+        </footer>  
+          
+    
```

Comment:
Removed some empty lines. Probably we also don't need the footer below, but not sure 
```suggestion
        <footer>
        </footer>  
```